modules/mediawiki/files/bin/mwscript.py

```diff
@@ -30,6 +35,8 @@
 print(command)
 confirm = input("Type 'Y' to confirm: ")
 if confirm.upper() == 'Y':
+    if long:
+        os.system(f'{command} (START)'})
     if 'generate' in locals():
         os.system(generate)
     return_value = os.system(command)
```

Comment:
Can we maybe add the time taken if `long = True` also?

Reply:
Yes we can capture time to execute